kapitan/dependency_manager/base.py

```diff
@@ -178,3 +189,56 @@ def fetch_http_source(source, save_dir):
     else:
         logger.warning("Dependency {} : failed to fetch".format(source))
         return None
+
+
+def fetch_helm_chart(dep_mapping, save_dir):
+    """
+    downloads a helm chart and its subcharts from source then untars and moves it to save_dir
+    """
+    lib = initialise_helm_fetch_binding()
+    unique_chart, deps = dep_mapping
+    for dep in deps:
+        chart_name = dep["chart_name"]
+        version = dep["version"]
+        repo_url = dep["source"]
+        destination_dir = dep["output_path"]
+
+        logger.info("Dependency helm chart {} and version {}: fetching now".format(chart_name, version))
+        c_chart_name = ffi.new("char[]", chart_name.encode("ascii"))
+        c_version = ffi.new("char[]", version.encode("ascii"))
+        c_repo_url = ffi.new("char[]", repo_url.encode("ascii"))
+        c_destination_dir = ffi.new("char[]", destination_dir.encode("ascii"))
+
+        res = lib.fetchHelmChart(c_repo_url, c_chart_name, c_version, c_destination_dir)
+        response = ffi.string(res).decode("utf-8")
+        if response != "":
+            logger.warning(
+                "Dependency helm chart {} and version {}: failed to fetch".format(chart_name, version)
+            )
+        else:
+            logger.info(
+                "Dependency helm chart {} and version {}: successfully fetched".format(chart_name, version)
+            )
+            logger.info(
+                "Dependency helm chart {} and version {}: saved to {}".format(
+                    chart_name, version, destination_dir
+                )
+            )
+
+
+def initialise_helm_fetch_binding():
+    """returns the dl_opened library (.so file) if exists, otherwise None"""
+    if platform.system() not in ("Linux", "Darwin"):  # TODO: later add binding for Mac
```

Comment:
Since Darwin is in this list, is this TODO comment outdated?

Reply:
My understanding is that the binding is still not available on Mac as of version `0.27.4`. I think that's what this PR from @yoshi-1224 is trying to achieve: https://github.com/deepmind/kapitan/pull/414

Reply:
Oh I see, I got confused with the actual comment and misread the if statement. :)